modules/wkt/src/hex-wkb-loader.ts

```diff
@@ -0,0 +1,61 @@
+// loaders.gl, MIT license
+
+import type {LoaderWithParser} from '@loaders.gl/loader-utils';
+import {BinaryGeometry} from '@loaders.gl/schema';
+
+import type {WKBLoaderOptions} from './wkb-loader';
+import {WKBLoader} from './wkb-loader';
+import {VERSION} from './lib/utils/version';
+import {decodeHex} from './lib/utils/hex-transcoder';
+
+export type HexWKBLoaderOptions = WKBLoaderOptions;
+
+/**
+ * Worker loader for Hex-encoded WKB (Well-Known Binary)
+ */
+export const HexWKBLoader: LoaderWithParser<BinaryGeometry, never, HexWKBLoaderOptions> = {
+  name: 'Hexadecimal WKB',
+  id: 'wkb',
+  module: 'wkt',
+  version: VERSION,
+  worker: true,
+  category: 'geometry',
+  extensions: ['wkb'],
+  mimeTypes: [],
+  options: WKBLoader.options,
+  text: true,
+  testText: isHexWKB,
+  // TODO - encoding here seems wasteful - extend hex transcoder?
+  parse: async (arrayBuffer: ArrayBuffer) => parseHexWKB(new TextDecoder().decode(arrayBuffer)),
+  parseTextSync: parseHexWKB
+};
+
+function parseHexWKB(text: string, options?: HexWKBLoaderOptions): BinaryGeometry {
+  const uint8Array = decodeHex(text);
```

Comment:
Should we call `isHexWKB()` to check the input string here?

Reply:
> Should we call `isHexWKB()` to check the input string here?

That would reduce the performance of parsing. The tradeoff that loaders.gl generally makes is that these parsers assume that the format is known. Separate test/testText fields exist on the loader to check, e.g. when selecting a loader from a set of supplied loaders, but we don't check again every time we parse.

That said, a review of error handling in general, and optional logging in case of parsing failures would be good topics for loaders.gl 5.0.